packages/did-core-test-server/suites/implementations/did-ethr-2021-consensys-mesh.json

```diff
@@ -0,0 +1,126 @@
+      "didResolutionMetadata":{
+        "contentType":"application/did+ld+json"
+      },
+      "didDocumentMetadata":{
+        "versionId": "12090175",
+        "updated": "2021-03-22T18:14:29Z"
+      }
+    }
+  },
+  "did:ethr:0x26bf14321004e770e7a8b080b7a526d8eed8b388?versionId=0": {
+    "didDocumentDataModel": {
+      "properties": {
+          "id": "did:ethr:0x26bf14321004e770e7a8b080b7a526d8eed8b388",
+          "verificationMethod": [
+            {
+              "id": "did:ethr:0x26bf14321004e770e7a8b080b7a526d8eed8b388#controller",
+              "type": "EcdsaSecp256k1RecoveryMethod2020",
+              "controller": "did:ethr:0x26bf14321004e770e7a8b080b7a526d8eed8b388",
+              "blockchainAccountId": "0x26bF14321004e770E7A8b080b7a526d8eed8b388@eip155:1"
+            }
+          ],
+          "authentication": [
+            "did:ethr:0x26bf14321004e770e7a8b080b7a526d8eed8b388#controller"
+          ],
+          "assertionMethod": [
+            "did:ethr:0x26bf14321004e770e7a8b080b7a526d8eed8b388#controller"
+          ]
+      }
+    },
+    "application/did+json": {
+      "didDocumentDataModel": {
+        "representationSpecificEntries": {
+        }
+      },
+      "representation": "{\"id\":\"did:ethr:0x26bf14321004e770e7a8b080b7a526d8eed8b388\",\"verificationMethod\":[{\"id\":\"did:ethr:0x26bf14321004e770e7a8b080b7a526d8eed8b388#controller\",\"type\":\"EcdsaSecp256k1RecoveryMethod2020\",\"controller\":\"did:ethr:0x26bf14321004e770e7a8b080b7a526d8eed8b388\",\"blockchainAccountId\":\"0x26bF14321004e770E7A8b080b7a526d8eed8b388@eip155:1\"}],\"authentication\":[\"did:ethr:0x26bf14321004e770e7a8b080b7a526d8eed8b388#controller\"],\"assertionMethod\":[\"did:ethr:0x26bf14321004e770e7a8b080b7a526d8eed8b388#controller\"]}",
+      "didResolutionMetadata":{
+        "contentType":"application/did+json"
+      },
+      "didDocumentMetadata":{
+        "nextVersionId": "12090175",
+        "nextUpdate": "2021-03-22T18:14:29Z"
+      }
+    },
+    "application/did+ld+json": {
+      "didDocumentDataModel":{
+        "representationSpecificEntries": {
+          "@context": [
+            "https://www.w3.org/ns/did/v1",
+            "https://identity.foundation/EcdsaSecp256k1RecoverySignature2020/lds-ecdsa-secp256k1-recovery2020-0.0.jsonld"
```

Comment:
We probably want to make this URL a w3id.org suite link, like the did key example above for consistency.